dmriprep/config/__init__.py

```diff
@@ -436,6 +436,11 @@ class workflow(_Config):
     ignore = None
     """Ignore particular steps for *dMRIPrep*."""
     longitudinal = False
+    """Number of ICA components to be estimated by MELODIC
+    (positive = exact, negative = maximum)."""
+    bold2t1w_dof = None
+    """Degrees of freedom of the BOLD-to-T1w registration steps."""
+    bold2t1w_init = "register"
```

Comment:
```suggestion
    bold2t1w_init = "register"
    """Whether to use standard coregistration ('register') or to initialize coregistration from the
    BOLD image-header ('header')."""
```
Although the PEP where attribute's docstrings were proposed was rejected, sphinx has maintained that style of documentation. The comment of the attribute goes AFTER the definition of the attribute.

Also, removed the dof option

Reply:
Thanks for catching this, I'll keep it in mind for the future.